lib/fetchHttpClient.ts

```diff
@@ -0,0 +1,210 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import "isomorphic-fetch";
+import * as tough from "tough-cookie";
+import AbortController from "abort-controller";
+import FormData from "form-data";
+
+import { HttpClient } from "./httpClient";
+import { WebResource } from "./webResource";
+import { HttpOperationResponse } from "./httpOperationResponse";
+import { HttpHeaders } from "./httpHeaders";
+import { RestError } from "./restError";
+import { Readable, Transform } from "stream";
+
+interface FetchError extends Error {
+    code: string | undefined;
```

Comment:
may be i am blind. Is the indentation 4 spaces? If so, then please change it to 2 spaces.

Reply:
I noticed you're using `code: string | undefined` instead of `code?: string`. is that because you want to require explicitly providing an `undefined`?